src/CMake/cpackLin.cmake

```diff
@@ -19,6 +19,9 @@ SET(CPACK_ARCHIVE_COMPONENT_INSTALL ON)
 SET(CPACK_DEB_COMPONENT_INSTALL ON)
 SET(CPACK_RPM_COMPONENT_INSTALL ON)
 
+# Force python byte compilation to use python3  
```

Comment:
I think your commit description is better than this comment line.
I don't really know what this line does by looking at it, but you explained it in the description.  Could you add more details here?